recipes/busybox/busybox-1.24.1/defconfig

```diff
@@ -783,18 +848,13 @@ CONFIG_FEATURE_IP_ROUTE=y
 # CONFIG_IPCALC is not set
 # CONFIG_FEATURE_IPCALC_FANCY is not set
 # CONFIG_FEATURE_IPCALC_LONG_OPTIONS is not set
-# CONFIG_NAMEIF is not set
-# CONFIG_FEATURE_NAMEIF_EXTENDED is not set
 CONFIG_NETSTAT=y
 # CONFIG_FEATURE_NETSTAT_WIDE is not set
 CONFIG_FEATURE_NETSTAT_PRG=y
 CONFIG_NSLOOKUP=y
 # CONFIG_NTPD is not set
 # CONFIG_FEATURE_NTPD_SERVER is not set
-CONFIG_PING=y
```

Comment:
You are disabling ping.  Probably a good idea, but...  It should go in separate commit, and you should provide a USE flag to enable it.  Or you or someone else should come up with a better plan for how to control this busybox spaghetti...